starter-code/src/app/components/my-movie-component/my-movie-component.component.html

```diff
@@ -0,0 +1,27 @@
+<h3>
+ {{movieDetails.title}}
```

Comment:
Now we know that at the beginning `movieDetails ` it will be undefined. Don't forget to handle this error with a `*ngIf`